core/domain/state_domain.py

```diff
@@ -3098,11 +3100,22 @@ def convert_html_fields_in_state(
                     state_dict['interaction']['default_outcome'],
                     conversion_fn))
 
+        if state_uses_old_rule_template_schema:
+            # We need to retrieve an older version of
+            # html_field_types_to_rule_specs to properly convert html, since
+            # past state schema v41, some html fields where removed.
```

Comment:
past --> after

(I misinterpreted this the first time I read it -- sounds like you were talking about a past schema)

Also, where --> were